src/js/utils/css.js

```diff
@@ -123,11 +120,34 @@ define([
         return style + '(' + channels.join(',') + ')';
     };
 
+    function toStyleString(styles) {
+        var styleString = '';
+
+        if (!styles) {
+            return '';
+        }
+
+        for (var name in styles) {
+            if (styles.hasOwnProperty(name)) {
+                styleString += _toSnakeCase(name) + ': ' + styles[name] + ';';
+            }
+        }
+
+        return '{' + styleString + '}';
+    }
+
+    function _toSnakeCase(name) {
+        return name.replace(/([A-Z])/g, function($1) {
+            return '-' + $1.toLowerCase();
+        });
+    }
+
     return {
         css: _css,
         style: _style,
         clearCss: styleLoader.clear,
         transform: transform,
-        hexToRgba: hexToRgba
+        hexToRgba: hexToRgba,
+        toStyleString: toStyleString
```

Comment:
Let's remove `toStyleString` asap. I don't want us adding additional util methods for something that the `css()` util should do.